src/xenia/kernel/xam/content_manager.cc

```diff
@@ -138,6 +147,59 @@ bool ContentManager::ContentExists(const ContentData& data) {
   return std::filesystem::exists(path);
 }
 
+X_RESULT ContentManager::CreateContentHeaderFile(const std::string file_name,
+                                                 const XCONTENT_DATA* data) {
+  auto title_id = fmt::format("{:8X}", kernel_state_->title_id());
+  auto header_path = root_path_ / title_id / kGameContentHeaderDirName;
```

Comment:
So this would put the save headers at eg. `content\{title_id}\Headers\{file_name}.header`, while the data itself would be in `content\{title_id}\{content_type}\{package_name}`?

Could see some problems coming up with that, eg. two contents with the same name but different content type would have header files collide, as unlikely as that is (doubtful games even use content types other than the SaveGame content type, but there's always a chance, maybe themes/avatar awards I guess)

Not really sure if storing header in a different location to the data itself is a great idea neither, would make save transfers/X360 conversions a bit more complicated... people have been used to the `title_id\content_type\file_name` format for years too.

I've always been a fan of putting header in parent folder of the data myself: https://github.com/xenia-canary/xenia-canary/blob/dea24e2fcb099c181f9617221205952880c048ee/src/xenia/kernel/xam/content_manager.cc#L100
Actually thinking about it, X360 already does do something similar with GOD files, header is stored at the normal package path while data files are put inside `{path}.data` folder next to it (also think GFWL also does something like that for DLC packages too, but it's been a while since I even used GFWL so I could be wrong about that :P), maybe it's worth using the same format for consistency.

Maybe this was already discussed and decided on though, just some thoughts on it.